src/sysfiles.c

```diff
@@ -1642,7 +1642,10 @@ Int SyGetch (
 UInt   syCTRO;                          /* number of '<ctr>-O' pending     */
 UInt   syESCN;                          /* number of '<Esc>-N' pending     */
 
+UInt FreezeStdin;    // Stop reading from stdin, this is used to stop HPC-GAP
```

Comment:
I'd change ", this" to ". This". I find it jarring to read like that (but perhaps that's my German background, commas trigger certain interpretations in my brain)